src/storage/storage.cc

```diff
@@ -337,15 +344,22 @@ Status Storage::Open(DBOpenMode mode) {
   propagate_opts.disable_auto_compactions = config_->rocks_db.disable_auto_compactions;
   SetBlobDB(&propagate_opts);
 
+  rocksdb::ColumnFamilyOptions lite_opts(subkey_opts);
+  if (config_->reduce_infrequent_column_memory_usage) {
```

Comment:
Here the `if` is useless if you delete the assignment for pubsub_opts and propagate_opts.

Reply:
update, please review